catalog/app/data/workflows.spec.js

```diff
@@ -0,0 +1,108 @@
+import parseWorkflows, {
+  emptyWorkflowsConfig,
+  workflowNotAvaliable,
+  workflowNotSelected,
+} from './workflows'
+
+describe('Workflows config', () => {
+  it('empty file', () => {
+    const config = parseWorkflows('')
+    expect(config).toBe(emptyWorkflowsConfig)
+    expect(config.workflows[0].slug).toBe(workflowNotAvaliable)
+  })
+
+  it('no workflows', () => {
+    const data = `
```

Comment:
we can use `dedent` module here to make these strings look a little nicer

Reply:
Maybe. But I think tests should have no extra stuff as much as possible.

Reply:
well i think readability is one of the most important aspects here, and one tiny extra dependency doesnt seem like a problem. i understand your point about avoiding adding complexity to the tests, but imo it's negligible in this case.